Description: Upstream changes introduced in version 0.1.0-2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 ctk (0.1.0-2) experimental; urgency=low
 .
   * Initial release (Closes: #601180)
 .
 The person named in the Author field signed this changelog entry.
Author: Dominique Belhachemi <domibel@debian.org>
Bug-Debian: http://bugs.debian.org/601180

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -57,6 +57,8 @@
 project(CTK)
 #-----------------------------------------------------------------------------
 
+ADD_SUBDIRECTORY(Utilities/Log4Qt)
+
 #-----------------------------------------------------------------------------
 # Library mode: SHARED (default) or STATIC
 #
@@ -140,10 +142,10 @@
   set(CTK_INSTALL_BIN_DIR "bin")
 endif()
 if(NOT CTK_INSTALL_LIB_DIR)
-  set(CTK_INSTALL_LIB_DIR "lib/ctk-${CTK_MAJOR_VERSION}.${CTK_MINOR_VERSION}")
+  set(CTK_INSTALL_LIB_DIR "lib")
 endif()
 if(NOT CTK_INSTALL_INCLUDE_DIR)
-  set(CTK_INSTALL_INCLUDE_DIR "include/ctk-${CTK_MAJOR_VERSION}.${CTK_MINOR_VERSION}")
+  set(CTK_INSTALL_INCLUDE_DIR "include")
 endif()
 if(NOT CTK_INSTALL_DOC_DIR)
   set(CTK_INSTALL_DOC_DIR "doc")
